mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -283,6 +283,7 @@ SONIC:
 	WithVoxelTurret:
 	Explodes:
 		RequiresCondition: !rank-elite
+		Weapon: UnitExplode
```

Comment:
You are changing the inherited value (from `^Vehicle` which is inherited by `^Tank`) here.

Reply:
I guess this falls under the "The following actors have been updated and might need manual adjustments" warning, so we can keep the update rule as it is and revert just this?